app/views/user_mailer/invite_email.html.erb

```diff
@@ -4,7 +4,7 @@
     </h1>
 <p>
     We are priviledged to give you an Invite to Beta Test Rank Hub, just follow the link below:
-    <%= url_for :controller => 'invites', :action => 'index', :hash=> @invite.hash ,:token => @str, :only_path => false, :host => 'https://rankhub-staging.herokuapp.com/' %>
+    <%= url_for :controller => 'invites', :action => 'index', :hash=> @invite.hash ,:token => @str, :only_path => false, :host => 'https://rankhub-staging.herokuapp.com' %>
```

Comment:
Looks like this PR was merged already.

@Nimmyjv Please create a new issue and fix this.